tests/unittests/core/utils/test_module_import.py

```diff
@@ -0,0 +1,58 @@
+"""Example usage and tests for :mod:`orion.core.utils.module_import`."""
+import pytest
+
+from orion.core.utils.module_import import ImportOptional
+
+
+class TestImportOptional:
+    def test_package_name_default_extra(self):
```

Comment:
Nitpick, I guess, but I'd be in favour of enabling pylint (and other code quality metrics/hooks) in tests as well, since we'd like those to be just as readable (if not even more) than our source code.
For instance, here, having a docstring on each test might be a little bit extra, but in general, I think it's a good idea to have some explanation of the desired behaviour.

I'm of the opinion that clear, well-written tests are the best form of documentation, but others might not agree.

Reply:
I am hesitant. We used to have linters enabled on `tests/` too, but the tests often required patterns that outraged the linters. The tests docstrings also often ended up being a uninformative rewording of the test name, so we ended up removing the constraint and only using docstrings when we considered the test name not explicit enough.

I guess your comment implies that some of these test names are not clear enough. :) Which ones do you think would deserve a clearer explanation (if not all)?

Reply:
Nah it's not a big deal. Keep them as-is.